.circleci/Makefile

```diff
@@ -0,0 +1,21 @@
+#PROJECT ?= istio-testing
+VERSION ?= 0.4.4
+#HUB ?= gcr.io/
+
+PROJECT ?= costinm
+HUB ?=
+
+# Note: The build directory is the root of the istio/test-infra repository, not ./
+image:
+	docker build -t ${HUB}$(PROJECT)/proxy-builder:$(VERSION) -f Dockerfile .
+
+
+push:
+	gcloud docker -- push "${HUB}$(PROJECT)/proxy-builder:$(VERSION)"
```

Comment:
What does this do? Why do we need to push to gcloud ?

Reply:
Not needed, will remove. I was trying to find which is faster...